timemachine/cpp/src/barostat.hpp

```diff
@@ -30,11 +31,21 @@ template <typename RealType> class MonteCarloBarostat {
 
     int get_interval();
 
+    double get_volume_scaling();
+
+    void set_volume_scaling(const double scaling);
```

Comment:
nit: naming: maybe `{get,set}_volume_scale_factor` to better align with the terminology [here](http://docs.openmm.org/latest/userguide/theory/02_standard_forces.html#montecarlobarostat)?

Reply:
Changed in [5c39c88](https://github.com/proteneer/timemachine/pull/1152/commits/5c39c8893119ec43434328d333b41ae638a5cf05)